common/src/main/java/org/apache/drill/common/config/DrillConfig.java

```diff
@@ -44,26 +46,33 @@
 import com.google.common.collect.ImmutableList;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigRenderOptions;
 
 public final class DrillConfig extends NestedConfig{
-//  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillConfig.class);
+  private static final Logger logger = getLogger(DrillConfig.class);
```

Comment:
I don't have a strong opinion about this either way, but I do think it is something we should just decide and make sure we have a hard consensus around the way it should be done. I would like to see this change made in bulk if we decide to do it. Please do as Chris did with his proposal to make the loggers private and suggest the change on the mailing list. If we can come to a consensus that we want to change it, we can open up a JIRA at that time to make a bulk replacement. Or as we have done with the issue around privatizing them, we can refactor over time as people see the wrong convention in a file they are editing.